### DISCUSSION TITLE
Is it an issue or just me?

### DISCUSSION THREAD

**Discussion by jejukin** [DarkflameUniverse/DarkflameServer#1627] [General]
I had a working Darkflame server, but after updating it, I've run into something peculiar:

Whenever I build a rocket, the client is dropped back to the login screen.  Subsequently, that minifigure will be dropped back to the login screen a couple of seconds after viewing their models inventory or attempting to use a rocket to travel anywhere.

This happens on a restored database of my original game - characters are fine with existing rockets, but attempts to build a new one or to rebuild the existing one causes that character to experience the bug 100% of the time thereafter.

This also happens on a fresh install with a brand new database.  

I can restore my old database and play with the existing characters and their current rockets, but without being able to build a rocket, new characters can't progress.  I mean technically they can, but after they've built the rocket, they'll crash/drop to login screen anytime they try to use it or view their models inventory.

Edit: It's not just the rockets, building a race car does it, too

**Comment by aronwk-aaron** [comment]
https://github.com/DarkflameUniverse/DarkflameServer?tab=readme-ov-file#allowing-a-user-to-connect-to-your-server
Please ensure that you have followed these steps, specifically setting `UGCUSE3DSERVICES=7:0` in your boot.cfg

**Reply by Xiphoseer** [reply]
FWIW, the reason is the game tries to load a dynamic icon, sends out a http request and doesn't handle that failing when no UGC server is set up. The flag just turns that connection off entirely.

**Comment by jejukin** [comment]
Yeah, that did it.  Mine had 7:1, and making it a 0 resolved this issue.  Thanks!
